apps/client/src/app/shop/[shop_id]/_components/ShopProductSection/ShopProductSection.tsx

```diff
@@ -1,13 +1,19 @@
 import { Flex, Grid } from "@chakra-ui/react";
 
 import { ShopProduct } from "../ShopProduct";
+import { useGetProductsByShop } from "../../_hooks";
 
 export const ShopProductSection = () => {
+  const productList = useGetProductsByShop();
+
   return (
     <>
       <Flex mt="32px">브랜드 상품</Flex>
       <Grid gridTemplateColumns="172px 172px" gap="24px 12px" mt="16px">
-        <ShopProduct id={0} imageUrl="https://picsum.photos/200" topCategory="" introduce="" price={0} />
+        {productList &&
+          productList.map((product) => {
+            return <ShopProduct key={product.name + product.id} {...product} />;
+          })}
```

Comment:
```suggestion
        {productList && productList.map((product) => <ShopProduct key={product.name + product.id} {...product} />)}
```

return 키워드를 굳이 사용할 필요가 있을까요??

Reply:
줄바꿈 때문에 return 사용했었는데 옵셔널체이닝으로 하니 똑같이 작성되네요! 수정했습니다 :)